quiche/src/tls.rs

```diff
@@ -1022,7 +1022,10 @@ extern fn new_session(ssl: *mut SSL, session: *mut SSL_SESSION) -> c_int {
 fn map_result(bssl_result: c_int) -> Result<()> {
     match bssl_result {
         1 => Ok(()),
-        _ => Err(Error::TlsFail),
+        _ => {
+            log_ssl_error();
```

Comment:
when testing this, I used a .crt instead of .pem by accident, and was very confused why I got TlsFail. This makes it a little easier to debug.